tests/test_data_utils.py

```diff
@@ -0,0 +1,169 @@
+import pandas as pd
+import pytest
+
+from vivarium_census_prl_synth_pop.constants import metadata
+from vivarium_census_prl_synth_pop.results_processing.formatter import (
+    combine_joint_filers,
+    flatten_data,
+    format_1040_dataset,
+)
+
+
+@pytest.fixture(scope="module")
+def dummy_1040():
+    return pd.DataFrame(
+        {
+            "simulant_id": list(range(8)) * 2,
+            "joint_filer": [
+                False,
+                True,
+                False,
+                False,
+                True,
+                False,
+                False,
+                False,
+            ]
+            * 2,
+            "relationship_to_reference_person": [
+                "Reference person",
+                "Opp-sex spouse",
+                "Biological child",
+                "Reference person",
+                "Opp-sex spouse",
```

Comment:
same-sex spouse?

Reply:
or maybe even make totally generic to avoid convusion and unnecessary thinking, e.g. "relationship 1", "relationship 2"